### DISCUSSION TITLE
Can I completely manage my own instance of Hyperswitch or do i need to connect to some central API?

### DISCUSSION THREAD

**Discussion by leonaburime** [juspay/hyperswitch#2990] [General]
Forgive me if this is an obvious question but I'm looking at several payment methods/packages for a startup and would like to go with something open source for cost and updates.

Can I manage my own instances **completely** without paying 4 cents per transaction or do I need to connect **and pay** to some hub for worldwide payments since it seems Hyperswitch is a company trying to make money from this product?

**Comment by manojradhakrishnan** [comment]
Hyperswitch is fully open source and you can deploy it on your own instance. Please refer the docs to self deploy here: https://opensource.hyperswitch.io/

The 4 cent plan is for the Hyperswitch Cloud offering.
